spec/features/check_answers_page_spec.rb

```diff
@@ -10,7 +10,7 @@
       click_on "Save and continue"
       fill_in "Benefit type", with: "Child benefit"
       fill_in "Enter amount", with: "150"
-      click_checkbox("benefit-model-benefit-frequency", :weekly)
+      choose "Every week"
```

Comment:
The drawback with this is that is doesn't work in a browser :-(

Reply:
As discussed on slack, fixing this by adding config that makes `choose` work in browser tests.